app/scss/_variables.scss

```diff
@@ -11,8 +11,8 @@ $mint: #9EBCC0;
 $teal: #01333B;
 $orange: #C83600;
 $mustard: #D09E4B;
-$yellow: #FDDF79;
-$pond: #1473E5;
+$peacock: #00A6BF; // 0, 166, 191
```

Comment:
This is fine, but maybe make an "enhancement" issue for the future for an elegant way to handle converting to (or just supplying additional variables for) RGB values?

Reply:
I think if we are only using it one time, it's fine.